scripts/how_to/adding_new_ida_to_toggle_report.rst

```diff
@@ -0,0 +1,135 @@
+=====================================
+Enabling Toggles Report for a new IDA
+=====================================
+
+The generation of toggles report for an IDA has been automated. However, for now, the automation has only been used on lms. This document lists what needs to be done to automate toggle report generation for other IDAs.
+
+.. contents:: Steps
+
+Gathering Data
+==============
+
+Annotation Data
+---------------
+The toggles are should be annotated in code following edX code annotation:`writing annotations`_. The `code_annotations`_ tool is used to collect annotations into a report.
```

Comment:
```suggestion
The toggles are annotated in code following edX code annotation:`writing annotations`_. The `code_annotations`_ tool is used to collect annotations into a report.
```

Reply:
Do we have some documentation on what annotations should be included for each toggle? I didn't see anything about that in the "writing annotations" link.

Reply:
this is something on my todo list to check